libopenage/renderer/stages/camera/manager.h

```diff
@@ -143,6 +144,17 @@ class CameraManager {
 	 * Uniform buffer input for the camera.
 	 */
 	std::shared_ptr<renderer::UniformBufferInput> uniforms;
+
+	/**
+	 * x and z bounds for the camera.
+	 */
+	std::pair<float, float> x_bounds, z_bounds;
+
+	/**
+	 * Constant values for the camera bounds.
+	 * TODO: Make boundaries dynamic based on map size.
+	 */
+	const float XMIN = 12.25f, XMAX = 32.25f, ZMIN = -8.25f, ZMAX = 12.25f;
```

Comment:
You could move these to https://github.com/SFTtech/openage/blob/master/libopenage/renderer/camera/definitions.h but since they are meant only for testing, this doesn't really matter.